taskcluster/app_services_taskgraph/build_config.py

```diff
@@ -10,20 +10,45 @@
 from taskgraph.util.memoize import memoize
 
 
+EXTENSIONS = {
+    'aar': ('.aar', '.pom', '-sources.jar'),
+    'jar': ('.jar', '.pom')
+
+}
+CHECKSUMS_EXTENSIONS = ('.sha1', '.md5')
+
+
 def get_components():
     build_config = _read_build_config()
     return [{
-        'version': build_config['libraryVersion'],
```

Comment:
`version`, `artifactId` and `publications` are unused as of now in the tasks' `attributes` hence I removed them to pass less data around. Will enrich the data struct once I deal with beetmover tasks, if needed.